clippy_lints/src/utils/conf.rs

```diff
@@ -15,10 +15,10 @@ pub struct Rename {
     pub rename: String,
 }
 
-/// A single disallowed method, used by the `DISALLOWED_METHOD` lint.
+/// A single disallowed method, used by the `DISALLOWED_METHODS` lint.
 #[derive(Clone, Debug, Deserialize)]
 #[serde(untagged)]
-pub enum DisallowedMethod {
+pub enum DisallowedMethods {
```

Comment:
I think this type should still be singular as it represents a single method

Reply:
Oh right. I'll revert this. Good catch!